OracleDatabase/18.4.0-XE/Vagrantfile

```diff
@@ -150,6 +157,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision "shell", path: "scripts/install.sh", env:
     {
        "SYSTEM_TIMEZONE"     => VM_SYSTEM_TIMEZONE,
+       "KEEP_DB_INSTALLER"   => VM_KEEP_DB_INSTALLER,
```

Comment:
FYI -- while this is correct, you need to keep in mind that `VM_KEEP_DB_INSTALLER` is a boolean not a string.
We have implicit conversion as Ruby booleans are objects implementing the `to_s()` method.

Reply:
@AmedeeBulle Understood. Thanks!